demo/src/app.html

```diff
@@ -27,6 +27,16 @@
 		</script>
 		<link rel="preload" href="https://www.agnosui.dev/versions.json" as="fetch" crossorigin="anonymous" />
 	</head>
+	<!-- Google tag (gtag.js) -->
+	<script async src="https://www.googletagmanager.com/gtag/js?id=G-49LCWGVXJ9"></script>
+	<script>
+		window.dataLayer = window.dataLayer || [];
+		function gtag() {
+			dataLayer.push(arguments);
+		}
+		gtag('js', new Date());
+		gtag('config', 'G-49LCWGVXJ9');
+	</script>
```

Comment:
As discussed, Firefox does not like having a script between the head and body, it does not load it:

![image](https://github.com/user-attachments/assets/a8b1af70-39df-4df0-aab1-ab3d250ee09a)